src/isMemberOf.ts

```diff
@@ -0,0 +1,41 @@
+import github from '@actions/github';
```

Comment:
```suggestion
import * as github from '@actions/github';
```

There doesn't seem to be a default export for `@actions/github`; GitHub actions was complaining that `github` was undefined in my testing.